guide/message-components/buttons.md

```diff
@@ -0,0 +1,172 @@
+			components: [row],
+		});
+	},
+};
+```
+
+Restart your bot and then send the command to a channel your bot has access to. If all goes well, you should see something like this:
+
+<DiscordMessages>
+	<DiscordMessage profile="bot">
+		<template #interactions>
+			<DiscordInteraction profile="user" :command="true">ban</DiscordInteraction>
+		</template>
+		Are you sure you want to ban <DiscordMention :highlight="true" profile="user" /> for reason: trolling?
+		<template #actions>
+			<DiscordButtons>
+				<DiscordButton type="secondary">Cancel</DiscordButton>
+				<DiscordButton type="danger">Confirm Ban</DiscordButton>
+			</DiscordButtons>
+		</template>
+	</DiscordMessage>
+</DiscordMessages>
+
+## Button styles
+
+You'll notice in the above example that two different styles of buttons have been used, the grey Secondary style and the red Danger style. These were chosen specifically to support good UI/UX principles. In total, there are five button styles that can be used as appropriate to the action of the button:
+
+<DiscordMessages>
+	<DiscordMessage profile="bot">
+		<template #interactions>
+			<DiscordInteraction profile="user" :command="true">buttons</DiscordInteraction>
+		</template>
+		<template #actions>
+			<DiscordButtons>
+				<DiscordButton type="primary">Primary</DiscordButton>
+				<DiscordButton type="secondary">Secondary</DiscordButton>
+				<DiscordButton type="success">Success</DiscordButton>
+				<DiscordButton type="danger">Danger</DiscordButton>
+				<DiscordButton type="link" url="https://discord.js.org">Link</DiscordButton>
+			</DiscordButtons>
+		</template>
+	</DiscordMessage>
+</DiscordMessages>
+
+- `Primary` style buttons are blue. These are suitable for most general purpose actions, where it's the primary or most significant action expected.
+- `Secondary` style buttons are grey. Use these for less important actions like the "Cancel" button in the example above.
+- `Success` style buttons are green. Similar to the Primary button, these are a good choice for "positive" confirmation actions.
+- `Danger` style buttons are red. Where the action being confirmed is "destructive", such a ban or delete, using a red button helps alert the user to the risk of the action.
+- `Link` style buttons are also grey, but are tagged with the "external link" symbol. These buttons will open the provided link in the browser without sending an interaction to the bot.
+
+## Link buttons
+
+Link buttons are a little different to the other styles. `Link` buttons _must_ have a `url`, _cannot_ have a `customId` and _do not_ send an interaction event when clicked.
+
+```js{3}
+const button = new ButtonBuilder()
+	.setLabel('discord.js docs')
+	.setURL('https://discord.js.org')
+	.setStyle(ButtonStyle.Link);
+```
+
+## Disabled buttons
+
+If you want to prevent a button from being used, but not remove it from the message, you can disable it with the <DocsLink section="builders" path="class/ButtonBuilder?scrollTo=setDisabled"/> method:
```

Comment:
```suggestion
If you want to prevent a button from being used, but not remove it from the message, you can disable it with the <DocsLink section="builders" path="ButtonBuilder:Class#setDisabled" type="method"/> method:
```